src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs

```diff
@@ -241,7 +241,7 @@ public async Task GivenAnInputFolderUnpackedFromBaseLayer_WhenPushOciFiles_IfUse
             string command = $"pull {testPushMultiLayersImageReference} -o checkMultiLayersFolder";
             await OrasClient.OrasExecutionAsync(command, Directory.GetCurrentDirectory());
             Assert.Equal(2, Directory.EnumerateFiles("checkMultiLayersFolder", "*.tar.gz", SearchOption.AllDirectories).Count());
-            Assert.Equal(4, StreamUtility.DecompressTarGzStream(File.OpenRead(Path.Combine("checkMultiLayersFolder", "layer2.tar.gz"))).Count());
+            Assert.Equal(4, StreamUtility.DecompressFromTarGzStream(new MemoryStream(File.ReadAllBytes(Path.Combine("checkMultiLayersFolder", "layer2.tar.gz")))).Count());
```

Comment:
I'm wondering what's the consideration of using memory stream instead of the file stream provided by the system?

Reply:
Fixed it in DecompressFromTarGz function to support filestream.

Reply:
Thanks, fixed DecompressFromTarGz() to support filestream.